test/api/unit/services/SQS.test.js

```diff
@@ -80,8 +80,6 @@ describe('SQS', () => {
     context("building a site's default branch", () => {
       it('should set an empty string for BASEURL in the message for a site with a custom domain', (done) => {
         const domains = [
-          'www.example.com',
```

Comment:
How can those sites work without a protocol? I must be missing something.

Like if a site currently has `boop.com` in their `domain` field, the "View Website" button currently links to `https://federalist-staging.18f.gov/sites/53/boop.com`, which doesn't work.